functions/tokenservice/functions/index.js

```diff
@@ -149,21 +149,17 @@ exports.getOAuthToken = functions.https.onCall(async (data, context) => {
   // Retrieve the token from the database
   const docRef = db.collection('ShortLivedAuthTokens').doc('OauthToken');
 
-  const doc = await docRef.get();
-
   try {
+    const doc = await docRef.get();
     if (doc.exists && isValid(doc.data().expireTime)) {
-      //push notification
       pushNotification(
         data['deviceID'],
         doc.data().accessToken,
         doc.data().expireTime
       );
-      return doc.data();
+      await doc.data();
     } else {
       const result = await retrieveCredentials(context);
-      console.log('Print result from retrieveCredentials functions');
```

Comment:
@santhoshvaddi I assume this deletion is fine with you?

Reply:
Yes, this is fine with me.